### DISCUSSION TITLE
Visualize basin time variables

### DISCUSSION THREAD

**Discussion by jc-hunink** [Deltares/Ribasim#890] [Q&A] [answered]
With the QGIS plugin i can visualize the model output but is it also possible to visualize the model input of the basis time variables (like precipitation, drainage, etc) in QGIS?

**Comment by Huite** [comment, marked as answer]
Hi Joachim,

Not in an easy way. We would probably require a specific plotting window to do so: https://github.com/Deltares/Ribasim/issues/790

(Outside of QGIS, the pivot table functionality in pandas is quite convenient here. It's shown in some of the examples.)
